cms/admin/permissionadmin.py

```diff
@@ -51,8 +51,9 @@ def get_queryset(self, request):
         # can see only permissions for users which are under him in tree
 
         # here an exception can be thrown
+        model = self.model
```

Comment:
Yes I think the last 2 ones are in the PR.

I’ve done and committed the adjustments you asked for now !

cheers

> Le 28 juil. 2015 à 08:31, Iacopo Spalletti <notifications@github.com> a écrit :
> 
> In cms/admin/permissionadmin.py <https://github.com/divio/django-cms/pull/4316#discussion_r35617625>:
> 
> > @@ -51,8 +51,9 @@ def get_queryset(self, request):
> >          # can see only permissions for users which are under him in tree
> >  
> >          # here an exception can be thrown
> > +        model = self.model
> Ok for settingsadmin.py:42
> I can't find permissionadmin.py:54, permissionadmin.py:117: aren't those in this PR?
> Use the qs = self.model... syntax everywhere in this PR, we will change settingsadmin later
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/divio/django-cms/pull/4316/files#r35617625>.
>